spec/features/multiwiki_assignment_spec.rb

```diff
@@ -89,7 +89,7 @@
       click_button 'Assign Articles'
       click_button 'Assign an article'
       within('#users') do
-        first('input').set('https://wikisource.org/wiki/Wp/kiu/Heyder_Cansa')
```

Comment:
This test with the wikisource url should stay in place, but we can add an additional test for incubator.